service/src/main/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderService.java

```diff
@@ -148,6 +164,20 @@ public String getMostRecentUploadedOrderDocumentUrl(final List<Element<Generated
             .getDocument().getBinaryUrl();
     }
 
+    public CaseDetails removeOrderProperties(CaseDetails caseDetails) {
+        caseDetails.getData().remove("epoRemovalAddress");
```

Comment:
It would probably be worth while creating an enum with these keys in them, nicer to only need to change it in one place than multiple if one of the id's change

Reply:
Would then be able to loop over the enum values and just remove everything a bit cleaner